src/routers/report.js

```diff
@@ -0,0 +1,21 @@
+const {Router} = require('express');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {Router} = require("express");
```